prbt_hardware_support/include/prbt_hardware_support/brake_test_utils.h

```diff
@@ -26,15 +26,14 @@
 
 #include <prbt_hardware_support/brake_test_utils_exception.h>
 
-static const std::string TOPIC_NAME{"/prbt/joint_states"};
-static constexpr double JOINT_STATES_COMPARISON_FREQUENCY{10.0};
-static constexpr double JOINT_STATES_TOPIC_TIMEOUT{1.0};
-static constexpr double DEFAULT_JOINT_STATES_COMPARISON_TOLERANCE{0.001};
-static constexpr double DEFAULT_ROBOT_MOTION_TIMEOUT_S{1.0};
+static const std::string TOPIC_NAME{ "/prbt/joint_states" };
```

Comment:
I see no reason for these spaces. What do you think? I would recommend removing this formatting rule.

Reply:
Maybe they are supposed to increase readability. But you are right - I will try to disable them.

Reply:
i can not find an option that changes this ... http://releases.llvm.org/5.0.0/tools/clang/docs/ClangFormatStyleOptions.html